docs/how-to-upload-manifest.md

```diff
@@ -1,107 +1,72 @@
 
-* Execute `login.sh` by providing your `quay.io` account user name and password as arguments
 ```bash
-$ chmod a+x login.sh
-$ ./login.sh {username} {password}
+$ ./scripts/get-quay-token
 ```
 
-Upon successful invocation, you will receive a JSON response with an access token, as shown below. Save the value of the `token` field from the JSON response. We will use this token to make calls to `quay.io` API. 
+You will be prompted to provide your `quay.io` account user name and password.
+
+Upon successful invocation, you will receive a JSON response with an access token, as shown below. Save the value of the `token` field from the JSON response as we will be using this token to make calls to `quay.io` API.
+
 ```json
 {
     "token": "basic XWtgc2hlbTpsZWR6ZXCwbYlf"
 }
 ```
 
-## Push Operator Manifest
-* Save the manifest into a file, for example `myapp.yaml`
-* Copy the following excerpt and save it into a file, for example `push.sh`. Put appropriate values for the variables.
-
-```bash
-#!/bin/sh
-
-FILENAME="file name (without the extension)"
-NAMESPACE="namespace in quay.io"
-REPOSITORY="repository name in quay.io"
-RELEASE="version/release of the operator"
-TOKEN="basic {access token here}"
+## Push An Operator Artifact
 
-function cleanup() {
-    rm -f ${FILENAME}.tar.gz
-}
-trap cleanup EXIT
+You can now push an operator artifact to `quay.io` by using the following bash script:
 
-tar czf ${FILENAME}.tar.gz ${FILENAME}.yaml
+```bash
+$ ./scripts/push-to-quay
+```
 
-BLOB=$(cat ${FILENAME}.tar.gz | base64 -w 0)
+You will be prompted for the path to the file containing your operator artifact, the namespace, repository, release and token related to your `quay.io` registry. The repository will be created automatically if it doesn't exist.
 
-curl -H "Content-Type: application/json" \
-     -H "Authorization: ${TOKEN}" \
-     -XPOST https://quay.io/cnr/api/v1/packages/${NAMESPACE}/${REPOSITORY} -d '
-{
-    "blob": "'"${BLOB}"'",
-    "release": "'"${RELEASE}"'",
-    "media_type": "helm"
-}'
-```
+For example, if you have saved the operator artifact into a file named `myapp.yaml` in the root if this repository and want to push it to `myoperators/myapp:1.0.0` then the inputs will have the following values:
 
-If you have saved the manifest into `myapp.yaml` and want to push it to `myoperators/myapp:1.0.0` then the variables will have the following values.
 ```bash
-FILENAME="myapp"
-NAMESPACE="myoperators"
-REPOSITORY="myapp"
-RELEASE="1.0.0"
+Relative path to your operator artifact file: myapp.yaml
+Namespace in quay.io: myoperators
+Repository name in quay.io: myapp
+Version/Release of operator: 1.0.0
+Quay.io token (TOKEN value of ./scripts/get-quay-token ): basic XWtgc2hlbTpsZWR6ZXCwbYlf
 ```
 
-* Execute the script 
-```bash
-$ chmod a+x push.sh
-$ ./push.sh
-```
+You should now be able to visit `quay.io` and browse the uploaded operator artifact in your desired namespace.
```

Comment:
It will also help to add  verbiage here too mentioning that you can have multiple operator offerings in example namespace and how they will be visible on quay.io. Call out that you won't be able to see the artifact you uploaded.